src/Driver.cpp

```diff
@@ -1,55 +1,56 @@
 #include <motors_weg_cvw300/Driver.hpp>
+#include <base/Angle.hpp>
 
 using namespace std;
 using namespace base;
 using namespace motors_weg_cvw300;
 
 Driver::Driver(int address)
     : m_address(address) {
+    // default of 7ms is too low for the weg controller at 57600
+    setInterframeDelay(base::Time::fromMilliseconds(14));
```

Comment:
Maybe make this a parameter?

Reply:
It actually is (`setInterframeDelay` is accessible). This sets a default that works for the WEG controller.